source/Octopus.Server.Client/Editors/Async/ProjectEditor.cs

```diff
@@ -65,7 +66,10 @@ public async Task<ProjectEditor> CreateOrModify(string name, ProjectGroupResourc
             return this;
         }
 
-        public async Task<ProjectEditor> CreateOrModify(string name, ProjectGroupResource projectGroup, LifecycleResource lifecycle, string description, string cloneId = null)
+        public async Task<ProjectEditor> CreateOrModify(string name, ProjectGroupResource projectGroup, LifecycleResource lifecycle, string description, string cloneId = null) => await CreateOrModify(name, projectGroup, lifecycle, description, cloneId, CancellationToken.None);
+        public async Task<ProjectEditor> CreateOrModify(string name, ProjectGroupResource projectGroup, LifecycleResource lifecycle, string description, CancellationToken cancellationToken) => await CreateOrModify(name, projectGroup, lifecycle, description, cloneId: null, cancellationToken);
+        
+        public async Task<ProjectEditor> CreateOrModify(string name, ProjectGroupResource projectGroup, LifecycleResource lifecycle, string description, string cloneId, CancellationToken cancellationToken)
```

Comment:
Decided to expand with overload, instead of `cancellationToken = default`. Even though there is no `interface` contract for this function, the `ProjectEditor` is a public class, with potential direct reference / usage.